nntrainer/utils/util_func.h

```diff
@@ -271,6 +275,16 @@ template <typename T, typename C = int> T enum_class_or(T e1, T e2) {
   return static_cast<T>(i1 | i2);
 }
 
+/**
+ * @brief Convert a relative path into an absolute path.
+ *
+ * @param __name relative path
+ * @param __resolved variable to store the result value.
+ *
+ * @return absolute path
+ */
+char *getRealpath(const char *__restrict __name, char *__restrict __resolved);
```

Comment:
Should we use '__' prefix for parameter name ?

Reply:
Thank you for your good point. In fact, it just followed definition of the original realpath function at stdlib. However, It would be better to remove the underbar from the parameter name because it is a custom function. I will reflect that modification. Thanks a lot!